app/code/Magento/MediaGalleryApi/Api/IsPathBlacklistedInterface.php

```diff
@@ -8,9 +8,11 @@
 namespace Magento\MediaGalleryApi\Api;
 
 /**
- * Check if the path is blacklisted for media gallery.
+ * Check if the path is excluded for media gallery.
  *
- * Directory path may be blacklisted if it's reserved by the system.
+ * Directory path may be excluded if it's reserved by the system.
+ * @deprecated Replaced due to abusive word usage
+ * @see \Magento\MediaGalleryApi\Api\IsPathExcludedInterface
  * @api
  */
 interface IsPathBlacklistedInterface
```

Comment:
@lenaorobei  Renaming `@api` interfaces is a MAJOR change.

Reply:
This interface was introduced in `2.4.x` release line and is not released yet.